app/src/main/java/org/permanent/permanent/ui/composeComponents/CustomDropdown.kt

```diff
@@ -88,11 +125,198 @@ fun CustomDropdown(isTablet: Boolean = false) {
                 modifier = Modifier.size(12.dp)
             )
         }
+
+        if (showBottomSheet) {
```

Comment:
Not sure if this should belong in the button or it should be in TypeSelection. For example, when you add a button to a page, that button only informs you that it was pressed and you take the decision based on that.

Reply:
Also, the ModalBottomSheet could be a separate page.